lib/date_time/date_time.c

```diff
@@ -263,8 +278,9 @@ static int current_time_check(void)
 		return -ENODATA;
 	}
 
-	if ((k_uptime_get() - time_aux.last_date_time_update) >
-	    CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS * 1000) {
+	if ((k_uptime_get() - time_aux.last_date_time_update) >=
```

Comment:
I had to add equality to this as my 30s interval test resulted to 30000ms delta ~70% of the time and the rest was 30001ms. Having equality here should be fine.

Reply:
@simensrostad Please check this and resolve this conversation if this is OK. Just want to point out what has been done after your approval.

Reply:
Looks ok.
Out-of-scope: It may be benficial if the library had a separate Kconfig option that specifies how old the internal date time reference can be before it can be updated. This is now decided by `CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS` which is slighly confusing.

Reply:
True, there should be a separate config for this. However, I don't even need any interval for my use case of this library so I'll leave this for future.